ui/app/abilities/token.js

```diff
@@ -0,0 +1,10 @@
+import AbstractAbility from './abstract';
+import { alias } from '@ember/object/computed';
+
+export default class extends AbstractAbility {
+  @alias('selfTokenIsManagement') canRead;
+  @alias('selfTokenIsManagement') canList;
+  @alias('selfTokenIsManagement') canWrite;
+  @alias('selfTokenIsManagement') canUpdate;
+  @alias('selfTokenIsManagement') canDestroy;
```

Comment:
`question`: Shouldn't these permissions be based on multiple properties?

Reply:
Token access and operations will only ever be available to Management tokens. If we want to have some kind of `policiesSupportTokenList` functionality later, it can be added to this file, but for now it's managers only — same as policies.